pkg/config/config.go

```diff
@@ -18,6 +18,7 @@
 package config
 
 import (
+	"github.com/apache/skywalking-eyes/pkg/comments"
```

Comment:
@dongzl Please move this to line 25 and this should fix your lint, try run `gofmt -s -w pkg/config/config.go` to fix it automatically. Just a formatting issue compaint by the CI.

Reply:
> @dongzl Please move this to line 25 and this should fix your lint, try run `gofmt -s -w pkg/config/config.go` to fix it automatically. Just a formatting issue compaint by the CI.

@Superskyyy  It's OK, Thanks very much.